### PR TITLE
made the changes in services.

### PR DESCRIPTION
**Pull request by vinodkumarc04**
Services and portfolio

### CODE REVIEW

**Review by ashok2world — APPROVED** [review]

File: src/components copy/Cards.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+const Cards = ({ title, iconPath, description, iconColor }) => {
+  return (
+    <div className="cardContainer">
+      <div className="card">
```
Comment (pooventhirankp27) [inline src/components copy/Cards.jsx]: Is this css required? don't see any import statement

File: src/components copy/Cards.jsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
+const Cards = ({ title, iconPath, description, iconColor }) => {
```
Comment (pooventhirankp27) [inline src/components copy/Cards.jsx]: import react first always

File: src/components copy/CardsContainer.jsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import Cards from "./Cards";
```
Comment (pooventhirankp27) [inline src/components copy/CardsContainer.jsx]: don't give folder name with white space

Comment (pooventhirankp27) [reply src/components copy/CardsContainer.jsx]: import react

File: src/components copy/CardsContainer.jsx [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+import { cardData } from "../utils/Constants";
+
+
+const CardsContainer = () => {
+  return (
+    <div className="card-container">
+      {cardData.map((card, i) => (
```
Comment (pooventhirankp27) [inline src/components copy/CardsContainer.jsx]: instead of i, keep it as key

File: src/components copy/ImageContainer.jsx [hunk 5]
```diff
@@ -0,0 +1,13 @@
+import { imagesData } from "../utils/Constants";
```
Comment (pooventhirankp27) [inline src/components copy/ImageContainer.jsx]: always import react. do this changes wherever applicable

File: src/components copy/Portfolio.jsx [hunk 6]
```diff
@@ -0,0 +1,11 @@
+import { ImageContainer } from "./ImageContainer";
+
+export function Portfolio() {
+  return (
+    <>
+      <h4 className="portfolio">PORTFOLIO</h4>
```
Comment (pooventhirankp27) [inline src/components copy/Portfolio.jsx]: Is this css required? don't see any import. import react first

File: src/components copy/Service.jsx [hunk 7]
```diff
@@ -0,0 +1,11 @@
+import CardsContainer from "./CardsContainer";
+
+export function Service() {
+  return (
+    <>
+      <h4 className="services">Services</h4>
```
Comment (pooventhirankp27) [inline src/components copy/Service.jsx]: Is this css required?

File: src/utils/Constants.jsx [hunk 8]
```diff
@@ -0,0 +1,92 @@
+export const cardData = [
```
Comment (pooventhirankp27) [inline src/utils/Constants.jsx]: keep this as .js file.

File: src/utils/Constants.jsx [hunk 9]
```diff
@@ -0,0 +1,92 @@
...
+      "M6.5 0a.5.5 0 0 0 0 1H7v1.07A7.001 7.001 0 0 0 8 16a7 7 0 0 0 5.29-11.584l.013-.012.354-.354.353.354a.5.5 0 1 0 .707-.707l-1.414-1.415a.5.5 0 1 0-.707.707l.354.354-.354.354-.012.012A6.97 6.97 0 0 0 9 2.071V1h.5a.5.5 0 0 0 0-1zm2 5.6V9a.5.5 0 0 1-.5.5H4.5a.5.5 0 0 1 0-1h3V5.6a.5.5 0 1 1 1 0",
+    description:
+      "Et harum quidem rerum facilis est et expedita distinctio. Nam libero tempore, cum soluta nobis est eligendi",
+    iconColor: "#4680ff",
+  },
+];
+
+
+
+export const imagesData = [
```
Comment (pooventhirankp27) [inline src/utils/Constants.jsx]: give more specific name. example, if portfolio images, it should be portfolioImages

File: src/index.css [hunk 10]
```diff
@@ -1,13 +1,149 @@
 body {
```
Comment (pooventhirankp27) [inline src/index.css]: don't use index.css file. inside your components create separate file and put the css code

**Review by pooventhirankp27 — CHANGES_REQUESTED** [review]

File: src/components copy/CardsContainer.jsx (hunk 3 above)
Comment (pooventhirankp27) [reply src/components copy/CardsContainer.jsx]: Add the logic inside components folder.

**Review by pooventhirankp27 — CHANGES_REQUESTED** [review]

File: src/index.css (hunk 10 above)
Comment (pooventhirankp27) [reply src/index.css]: don't commit index.css file

File: src/utils/Constants.js [hunk 11]
```diff
@@ -0,0 +1,92 @@
+export const ServiceCardData = [
```
Comment (pooventhirankp27) [inline src/utils/Constants.js:1]: keep it in camel case

File: src/App.js
Comment (pooventhirankp27) [inline src/App.js:1]: don't commit app.js file

File: src/components copy/ImageCard.jsx
Comment (pooventhirankp27) [inline src/components copy/ImageCard.jsx:1]: if this folder is not required, remove it.

**Review by pooventhirankp27 — CHANGES_REQUESTED** [review]

File: src/components copy/ImageCard.jsx
Comment (vinodkumarc04) [reply src/components copy/ImageCard.jsx:1]: it's not required in components copy folder so removed in my project

**Review by pooventhirankp27 — APPROVED** [review]

**Comment by pooventhirankp27** [comment]
Delete components copy folder in next PR

**Pull request merged**